File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/youtube/playerResponse/Thumbnail.java
package com.github.kotvertolet.youtubejextractor.models.youtube.playerResponse;

import java.io.Serializable;
import java.util.List;

public class Thumbnail implements Serializable {
    private List<ThumbnailsItem> thumbnails;

    public List<ThumbnailsItem> getThumbnails() {
        return thumbnails;
    }

    public void setThumbnails(List<ThumbnailsItem> thumbnails) {
        this.thumbnails = thumbnails;
    }

    @Override
    public String toString() {
        return
                "Thumbnail{" +
                        "thumbnails = '" + thumbnails + '\'' +
                        "}";
    }
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/Thumbnail.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Thumbnail : Parcelable {
    @SerializedName("thumbnails")
    var thumbnails: List<ThumbnailsItem>? = null
    override fun toString(): String {
        return "Thumbnail{" +
                "thumbnails = '" + thumbnails + '\'' +
                "}"
    }
}